Sources/CodeGen/CXX/CXXModule+Extensions.swift

```diff
@@ -412,14 +412,22 @@ extension CXXContinueStmt: Writeable {
 
 private struct AnyExpr: Writeable {
 
+  /// The C++ expression to write to the output C++ stream.
   let source: CXXExpr
 
-  init(_ source: CXXExpr) {
+  /// True if we need to write the expression in parentheses.
```

Comment:
```suggestion
  /// `true` iff we need to write the expression in parentheses.
```

Minor: only for consistency with other docs